jekyll/_posts/2021-07-21-Nim-Efficienct-Expressive-Elecgant-Benchmarking.md

```diff
@@ -0,0 +1,369 @@
+| title | author | excerpt |
+| :---: | :----: | :---: |
+| Benchmarking the Beast | W. Gordon Goodsman (GordonBGood) | use CPU caches efficiently |
```

Comment:
Can you also follow:

```
---
title: "Pattern matching in Nim"
author: haxscramper
excerpt: "Nim fusion and pattern matching"
---
```

No idea if the above will work

Reply:
Also, your excerpt needs to be longer. It does get shown on nim-lang.org main page so make it good.

Reply:
I'll make the excerpt more descriptive.

I don't know how the "Pattern matching" article might apply  to this simple benchmark but it might be able to be tied in to later articles in the series I propose.

Reply:
I only mean to use the same syntax for the front matter, I just copied it from one of the other articles.

Reply:
Okay, I'll change that part of the heading.